desktop/hardened-desktop/admin-controls/index.md

```diff
@@ -0,0 +1,57 @@
+---
+description: admin controls for desktop
+keywords: admin controls, rootless, docker desktop, hardened desktop
+title: What is Admin Controls?
+--- 
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+Admin Controls is a feature that gives administrators the ability to enforce certain Docker Desktop settings for their organization. 
+
+With a few lines of JSON, administrators are able to enforce preferences for HTTP proxies, network settings, and the Docker Engine configuration. This saves significant time and cost in securing developer workflows.
+
+### Who is it for? 
+
+- For Organizations who wish to configure Docker Desktop to be within their organization's centralized control.
+- For Organizations who want to create a standardized Docker Desktop environment at scale.
+- For security conscious Docker Business customers who want to confidently manage their use of Docker Desktop within tightly regulated environments.
+
+### What can be set?
+
+Using the `admin-settings.json` file, administrators can:
+
+- Switch on Enhanced Container Isolation
+- Configure HTTP Proxies
+- Configure network settings
+- Expose daemon on tcp://localhost:2375 without TLS Resources. This is applicable to Docker Desktop on Windows only.
+- Enforce the use of WSL2 based engine or Hyper-V
+- Configure Docker Engine
+- Turn off Docker Desktop's ability to checks for updates
+- Turn off Docker Desktop's ability to send usage statistics
+
+For more details on the syntax and options administrators can set, see [Configure Admin Controls](configure-ac.md).
+
+### How is this different to the `settings.json` file?
+
+Using the `settings.json` file to pre-configure Docker Desktop settings means that developers own the `settings.json` file. They can therefore adjust any settings that their admins create, for example, modifying network and proxy controls. 
```

Comment:
i'd change the first sentence to 'The settings.json is located in a directory that users can write to. As such, users can adjust any settings created by their admins in the settings.json, for example....'

Reply:
I would skip the "Expose daemon on tcp://localhost:2375" bullet in this list, because it actually reduces security of Docker Desktop.